### DISCUSSION TITLE
Remote repo support

### DISCUSSION THREAD

**Discussion by MHHenriksen** [community/community#3532] [Codespaces] [answered]
Will you support non-GitHub repos? We have a self-hosted GitLab server that would be fantastic to use with codespaces. I suppose mirroring to GitHub would be a solution, but we're not allowed to do that.

**Comment by PranamBhat** [comment]
Hi @MHHenriksen 

Not at the moment.

It would be a new feedback/feature request going forward.

You can bring it here : [https://github.community/](https://github.community/)

**Comment by MHHenriksen** [comment, marked as answer]
Thanks @PranamBhat, your link led me to these resources that have already answered my question:

https://github.community/t/self-hosted-codespaces/131361

https://docs.microsoft.com/en-us/visualstudio/codespaces/resources/consolidation-faq#im-currently-using-a-different-service-to-host-my-repositories-eg-azure-repos-bitbucket-etc-can-i-still-use-that-with-github-codespaces
